applications/source/rabbit-source/README.adoc

```diff
@@ -0,0 +1,103 @@
+//tag::ref-doc[]
+= RabbitMQ Source
+
+The "rabbit" source enables receiving messages from RabbitMQ.
+
+The queue(s) must exist before the stream is deployed; they are not created automatically.
+You can easily create a Queue using the RabbitMQ web UI.
+
+== Input
+
+N/A
+
+== Output
+
+=== Headers
+
+* `content-type: text/plain`
+
+=== Payload
+
+* `String`
+
+=== Headers
+
+* `content-type: application/octet-stream`
+
+=== Payload
+
+* `byte[]`
+
+=== Headers
+
+* `content-type: application/x-java-serialized-object`
+
+=== Payload
+
+* `java.io.Serializable`
```

Comment:
Also not true. I believe it is a `byte[]` since it is a JSON n content-type and we don't have any POJO type to convert such a payload into.